packages/storybook/src/web-components/button-login/button-login.md

```diff
@@ -0,0 +1,154 @@
+
+### Aanpasbare opmaak
+
+Gebruik voor de volgende design tokens waardes uit uw huisstijl.
+
+#### Font
+
+| Token                                 | Waarde                 |
+| :------------------------------------ | :--------------------- |
+| `--lux-button-login-font-family`      | `Rijksoverheid, Arial` |
+| `--lux-button-login-font-size`        | `18px`                 |
+| `--lux-button-login-font-line-height` | `24px`                 |
+
+#### Focus outline
+
+| Token                                     | Waarde    |
+| :---------------------------------------- | :-------- |
+| `--lux-button-login-focus-outline-color`  | `#282828` |
+| `--lux-button-login-focus-outline-offset` | `2px`     |
+| `--lux-button-login-focus-outline-style`  | `solid`   |
+| `--lux-button-login-focus-outline-width`  | `2px`     |
+
+### Optionele aanpasbare opmaak
+
+Gebruik voor de volgende design tokens de aangeboden waardes, of waardes uit uw huisstijl die vergelijkbaar zijn. Dit zorgt ervoor dat de inlogervaring zo herkenbaar mogelijk is bij de verschillende publieke dienstverleningen.
+
+#### Border
+
+| Token                              | Waarde |
+| :--------------------------------- | :----- |
+| `--lux-button-login-border-width`  | `1px`  |
+| `--lux-button-login-border-radius` | `4px`  |
+
+#### Color
+
+Denk aan de contrastvereisten van de WCAG.
+
+| Token                                        | Waarde    |
+| :------------------------------------------- | :-------- |
+| `--lux-button-login-background-color`        | `#282828` |
+| `--lux-button-login-border-color`            | `#696969` |
+| `--lux-button-login-color`                   | `#282828` |
+| `--lux-button-login-active-background-color` | `#a7a7a7` |
+| `--lux-button-login-active-border-color`     | `#282828` |
+| `--lux-button-login-active-color`            | `#282828` |
+| `--lux-button-login-hover-background-color`  | `#e6e6e6` |
+| `--lux-button-login-hover-border-color`      | `#696969` |
+| `--lux-button-login-hover-color`             | `#282828` |
+
+#### Size
+
+Schaal het icoon mee met de hoogte van de knop. Houd de knop minimaal 48px hoog in verband met de toegankelijke target size.
+
+| Token                                | Waarde  |
+| :----------------------------------- | :------ |
+| `--lux-button-login-min-inline-size` | `118px` |
+| `--lux-button-login-icon-size`       | `40px`  |
+
+## State
+
+<ul class="section">
+  <li>
+    <lux-button-login agent="digid"></lux-button-login>
+    <span>Default</span>
+  </li>
+  <li>
+    <lux-button-login class="sb-state--hover" agent="digid"></lux-button-login>
+    <span>Hover</span>
+  </li>
+  <li>
+    <lux-button-login class="sb-state--active" agent="digid"></lux-button-login>
+    <span>Active</span>
+  </li>
+  <li>
+    <lux-button-login class="sb-state--focus" agent="digid"></lux-button-login>
+    <span>Focus</span>
```

Comment:
Beetje off-topic maar we hebben nu ook [focus-visible](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible). Omdat we de visuele focus kernmerken juist hiervoor willen inzetten vermoed ik dat de tokens voor focus op den duur iets als 'prefix.button.**focus-visible**.etc.' zullen worden. Maargoed onze volgende stap om dit uit te werken was om dit bij Utrecht door te voeren, daar is nog het wachten op.

Reply:
In de code gebruik ik ook `:focus-visible`. Echter maakt dat voor onze gebruikers niet uit. In dit specifieke geval gaat het er om dat we het component in verschillende user states kunnen tonen in de documentatie. Zo ook met visual regression testing. Wat mij vertelt dat dat de `sb-` prefix niet toereikend is voor alle bedoelde gebruiken. Ik maak er `force-state--[state]` van.